jmx/jmx.go

```diff
@@ -20,13 +20,18 @@ import (
 	"github.com/newrelic/infra-integrations-sdk/log"
 )
 
+const (
+	jmxLineBuffer = 4 * 1024 * 1024 // Max 4MB per line. If single lines are outputting more JSON than that, we likely need smaller-scoped JMX queries
```

Comment:
you are right, that's the line limitation and we are not warning about not parsed lines

to avoid this we should remove the buffered IO, I have no idea if this behaviour was intended, but just in case it was not: https://github.com/newrelic/infra-integrations-sdk/pull/192